translations/translated/mudlet_en_US.ts

```diff
@@ -4,140 +4,90 @@
 <context>
     <name>GLWidget</name>
     <message numerus="yes">
-        <location filename="../../src/glwidget.cpp" line="383"/>
+        <location filename="../../src/glwidget.cpp" line="290"/>
         <source>You have a map loaded (%n room(s)), but Mudlet does not know where you are at the moment.</source>
         <translation>
-            <numerusform>You have a map loaded (one room), but Mudlet does not know where you are at the moment.</numerusform>
+            <numerusform>You have a map loaded (1 room), but Mudlet does not know where you are at the moment.</numerusform>
```

Comment:
The previous text said `one` for one room but `2` for two and so on. So the use of numbers/string needs to be fixed (IMHO). So this is wanted.